### PR TITLE
Fixing local weighted models confidence

### PR DESCRIPTION
**Pull request by mmerce**

### CODE REVIEW

**Comment by mmerce** [comment]
@sdesimone, could you please review the changes?

File: lib/Tree.js [hunk 1]
```diff
@@ -599,7 +599,7 @@ Tree.prototype.predict = function (inputData, path, missingStrategy, median,
     'path': path,
     'confidence': this.confidence,
     'distribution': this.distribution,
-    'count': getInstances(this.distribution),
+    'count': this.count,
```
Comment (sdesimone) [inline lib/Tree.js]: Just a quick check: it seems `getInstances` is not used anywhere in the project (anymore). So, how does the `this.count` property get initialised? I was expecting some kind of equivalent logic somewhere and it is not apparent to me what replaces `getInstances`... (but maybe I should look better). BTW, if it is correct that `getInstances` is not used anymore, maybe it is a good candidate for cleanup.

Comment (mmerce) [reply lib/Tree.js]: We've replaced getInstances with the count of instances in the node. The side effec is that weighted models will use this count to compute the associated confidence. You're right about the fact that getInstances is not used then anywhere, so we can get rid of it.

Comment (sdesimone) [reply lib/Tree.js]: ok, thanks -- so, the count now comes from the resource JSON, I understand.

Comment (mmerce) [reply lib/Tree.js]: yes, it's the original count of instances in the node

**Pull request merged**